helm/thingsboard/templates/node-db-configmap.yaml

```diff
@@ -51,7 +51,7 @@ data:
   SQL_ATTRIBUTES_BATCH_THREADS: {{ .Values.sql.attributes.batch.threads | quote }}
   SQL_EVENTS_BATCH_THREADS: {{ .Values.sql.events.batch.threads | quote }}
   SQL_TS_BATCH_THREADS: {{ .Values.sql.ts.batch.threads | quote }}
-  SQL_TS_LATEST_BATCH_THREADS: {{ .Values.sql.ts-latest.batch.threads | quote }}
-  QUEUE_RULEENGINE_PROMETHEUSSTATS_ENABLED: {{ .Values.rule-engine.prometheus-stats.enabled | quote }}
-  TB_QUEUE_RE_MAIN_PROCESSING_STRATEGY_TYPE: {{ .Values.rule-engine.queues.main.processing-strategy-type | quote }}
+  SQL_TS_LATEST_BATCH_THREADS: {{ .Values.sql "ts-latest" "batch" "threads" | quote }}
```

Comment:
Don't we need the "index" keyword, like [here](https://github.com/midokura/evp-chart/blob/767c558/charts/evp/templates/properties-configmap.yaml#L12)?

Reply:
good catch :muscle: